cupy/random/_generator.py

```diff
@@ -212,12 +212,14 @@ def geometric(self, p, size=None, dtype=int):
         p = cupy.asarray(p)
         if size is None:
             size = p.shape
+        if dtype is None:
+            dtype = numpy.dtype('l')
```

Comment:
Can we avoid creating a new dtype instance everytime? (or use `numpy.int_`?)

Reply:
Replaced with `numpy.long`. (`numpy.int_` is `q` on Windows)